### DISCUSSION TITLE
Improvement: half waveforms.

### DISCUSSION THREAD

**Discussion by janwirth** [samhirtarif/react-audio-visualize#15] [Ideas]
If the waveform is not mirrored it will be more space efficient.

<img width="911" alt="image" src="https://github.com/samhirtarif/react-audio-visualize/assets/7485959/959907a7-5410-4f2d-8b87-8c4fa2f95963">

**Comment by samhirtarif** [comment]
Moving this to the Ideas section in Discussions

**Comment by samhirtarif** [comment]
Are you suggesting that only the top half of the waveform should be displayed?
I believe this can be easily catered to with a prop, allowing for both mirrored and non-mirrored visualizations

**Reply by janwirth** [reply]
Either only top half or only bottom half.

```ts
type Symmetry = "Mirror" | "Top" | "Bottom"
```

**Reply by SalahAdDin** [reply]
@samhirtarif Did you add this feature?

**Reply by SalahAdDin** [reply]
@janwirth did you get this done on your own?

**Reply by SalahAdDin** [reply]
We are aiming to do something like this:
![image](https://github.com/user-attachments/assets/9c8cb81a-8713-4ff6-8f97-947b8ab664b6)

**Reply by janwirth** [reply]
No I didn't work on this.
